### PR TITLE
Feature/57 update minimized button behaviour remove nothing todo

### PR DESCRIPTION
**Pull request by JonasJurczok**

### CODE REVIEW

File: locale/fr/locale.cfg [hunk 1]
```diff
@@ -20,8 +20,7 @@ assign_self=Prendre
 show_done=Montrer les tâches finies
 hide_done=Masquer les tâches finies
 
-todo_maximize_button_caption=__1__: __2__
-nothing_todo=Rien à faire.
+tasks_available=__1__ Tasks available.
```
Comment (Tarrke) [inline locale/fr/locale.cfg]: Should be:
`tasks_available=__1__ tâches disponibles.`

File: src/todo/todo.lua [hunk 2]
```diff
@@ -143,18 +143,26 @@ function todo.update_current_task_label(player)
...
             todo.log(serpent.block(task))
             todo.get_maximize_button(player).caption =
-                {"todo.todo_maximize_button_caption",
+                {"",
                  {"todo.todo_list"},
+                 ": ",
                  string.match(task.task, "[^\r\n]+")}
             return
         end
+        count = i
```
Comment (Tarrke) [inline src/todo/todo.lua]: I think you should count only tasks with no assignee here.
`if (task.assignee == nil) then
  count = count + 1
end`

Or you should change the display from "Tasks available." to "Tasks to be done."

Comment (JonasJurczok) [reply src/todo/todo.lua]: hmmm... fair point. #66 :)

**Comment by Tarrke** [comment]
I did test that tonight and I'm pretty sure it's working as intended but
not as Perl regexp. So no worries about that.

**Comment by Tarrke** [comment]
I'll make thses corrections as soon as the creator/editor feature is merged.

**Pull request merged**